### DISCUSSION TITLE
stendhal

### DISCUSSION THREAD

**Discussion by coobbi** [coobbi/Jre4android#2] [General]
https://github.com/arianne/stendhal
run on Jre4android
![Uploading WechatIMG635.jpeg…]()

**Comment by coobbi** [comment]
![WechatIMG635](https://github.com/coobbi/Jre4android/assets/138000094/3c52bd3c-06b2-43b8-8c11-5e5a63958594)

**Comment by coobbi** [comment]
Setting->Classpath->add other lib/*.jar

**Comment by KingofGamer619** [comment]
Hello

**Comment by AntumDeluge** [comment]
Very interesting to see the Java client running on Android. But, it is recommended to use our official Android client ( https://f-droid.org/packages/org.stendhalgame.client ). It is a wrapper for our web client which will eventually replace the Java client.
